docs/README.md

```diff
@@ -0,0 +1,29 @@
+# Overview
+
+This document provides a basic overview to how js-ceramic is implemented. It goes though some of the high level apis of Ceramic and how it communicates with some external components. This is by no means comprehensive and will be expanded over time.
+
+## Ceramic Api
+
+CeramicApi is an interface that both CeramicCore and CeramicHttpClient implements. It provides all the methods needed by a developer to build on top of Ceramic. An instance of CeramicApi allows only one user to be set at a time, using the [DID](https://github.com/ceramicnetwork/js-did) interface. 
+
+![CeramicApi](/Users/oed/ceramic/js-ceramic/docs/images/ceramic-api.png)
```

Comment:
This image source is on your computer, not the one added in the PR

Reply:
ugh, that's annoying. Not sure why my md tool did that.